### DISCUSSION TITLE
Pause polling on certain zwave devices

### DISCUSSION THREAD

**Discussion by Rollem1** [zwave-js/node-red-contrib-zwave-js#169] [Q&A] [answered]
Hi All Happy New Year!

What is the best way to pause polling on a few Zwave devices? 

Background - I have some devices that are always in service but I also have some that I use to control Xmas Lights which are only plugged in for one month every year. When they are removed the controller is still looking for these so I'd like to now the best way to stop polling them without excluding them.

Thanks,
Jon

**Comment by marcus-j-davies** [comment]
Happy new year @Rollem1,

The driver doesn't poll as such, it only tries to ping them at every start up.

Sadly, there isn't much one can do here.
ZWave doesn't have a concept of 'temporary devices'

If these devices are mains powered, the driver expects them to respond (via a ping at start up).
After a while, the ping will fail (for obvious reasons) and they will be marked as DEAD - they will try and get pinged again at start up.

battery devices don't have this - as they are expected to not respond

With that said, I don't know if you can change the timeouts (Controller Node) to maybe speed up this timeout - allowing the init to happen quicker - but it could add problems for devices that are indeed alive, if these timeout are too small.

@AlCalzone - can changing timeouts actually make the ping return (or fail quicker?) - have never really played with these values.

**Comment by Rollem1** [comment]
Thanks @marcus-j-davies 

They are all mains powered. The Zwave console see's them as failed 
![image](https://user-images.githubusercontent.com/45107860/147936453-c7d93327-460f-405e-82db-7f939e948ebb.png)

but also debug console is filling up with these messages every minute or so
![image](https://user-images.githubusercontent.com/45107860/147936577-1defaff4-ef4a-41ad-927c-aa31ad844764.png)

Is there maybe a way to inhibit the error message?

**Reply by AlCalzone** [reply]
Like @marcus-j-davies said, the included devices are stored in the stick's memory and it might even expect them to exist for some tasks. So this might work just fine for you, but it might also have a negative effect on your network stability. Some nodes might be routing through these devices, so after removing or adding them, a network heal may be necessary.

The driver only pings listening devices once on startup to check if they actually listen or if they are dead. After that, no further communication with these nodes is done, unless you instruct the driver to do so. How long it takes for these pings to fail isn't something the driver has an influence on. It shouldn't be much of a problem though, unless you have many "ghost" devices and restart the driver often.

Z-Wave JS doesn't do polling. If these messages come up frequently, something in userland is trying to communicate with them.

That said, I might add a function to disable/ignore nodes at the driver level. This comes up every now and then and while I don't think it's a good idea, the alternative of excluding and re-including every now and then isn't very user friendly.

**Reply by marcus-j-davies** [reply]
Hi @Rollem1,

As mentioned above - a future update may see the ability to 'ignore' devices.
As for the error logs - its likely the affect of the UI trying to load / query them - their timestamps are almost next to each other.

so every time you open up the UI - it will try and query them - in fact it does, as the UI pull in a list, and that list involves querying devices for certain properties - unless you have some routine, trying to so something with them, if not - its just the UI expecting them be alive.

**Comment by Rollem1** [comment, marked as answer]
Thanks @AlCalzone @marcus-j-davies 

I found the culprit - I am using Bigtimer to switch these nodes on a schedule. It sends the "on" or "off" commands every minute. I thought I had stopped it by manually switching bigtimer to off. but this was sending off every minute. By sending "stop" to Bigtimer I have also managed to stop the errors in the debug console. Thanks for pointing me in the right direction to find this.
